pkg/provider/azure_vmss.go

```diff
@@ -852,15 +935,20 @@ func extractResourceGroupByVMSSNicID(nicID string) (string, error) {
 
 // GetPrimaryInterface gets machine primary network interface by node name and vmSet.
 func (ss *ScaleSet) GetPrimaryInterface(nodeName string) (network.Interface, error) {
-	managedByAS, err := ss.isNodeManagedByAvailabilitySet(nodeName, azcache.CacheReadTypeDefault)
```

Comment:
why changing cache type to unsafe here?

Reply:
A node management type (by uniform, flex or avset) cannot be changed after creation, so there is no need to fresh cache when checking it.